tests/bfd/test_bfd_static_route.py

```diff
@@ -218,8 +210,9 @@ def test_bfd_with_lc_reboot_ipv4(
             ),
         )
 
-    def test_bfd_static_route_deletion_ipv4(
+    def test_bfd_with_lc_reboot_ipv6(
```

Comment:
I adjusted the order of test cases so that the same test for IPv4 and IPv6 can sit next to each other.